apps/theming/lib/Listener/BeforePreferenceListener.php

```diff
@@ -25,23 +25,32 @@
  */
 namespace OCA\Theming\Listener;
 
+use OC\App\AppManager;
 use OCA\Theming\AppInfo\Application;
 use OCP\Config\BeforePreferenceDeletedEvent;
 use OCP\Config\BeforePreferenceSetEvent;
 use OCP\EventDispatcher\Event;
 use OCP\EventDispatcher\IEventListener;
 
 class BeforePreferenceListener implements IEventListener {
+	public function __construct(
+		private AppManager $appManager,
```

Comment:
```suggestion
		private IAppManager $appManager,
```
Only use OCP from applications